### DISCUSSION TITLE
Add return dislike function

### DISCUSSION THREAD

**Discussion by extraeme** [TeamVanced/VancedManager#752] [Ideas]
It would be cool if Vanced could include the works of https://returnyoutubedislike.com/ and include a visible dislike counter through this service.

**Comment by SaranSDS008** [comment]
Yeah plz bring this feature. I would love to see this feature on Vanced Youtube app (I use it on Desktop site though).

**Comment by Repiteo** [comment]
Echoing this, it'd be a very welcome feature!

**Comment by GitStudying** [comment]
I'm not sure when if it isn't already but the dislike function will be completely removed from the YouTube API and therefore is not worth putting effort in right now in my opinion.
